### PR TITLE
WinUI-2902 : Added the AnimationType in the BusyIndicator control.

### PR DESCRIPTION
**Pull request by Priyanka-Vijayasankar**
### Description:
Prepared AnimationType MD file in BusyIndicator control.

### CODE REVIEW

File: winui/BusyIndicator/AnimationType.md [hunk 1]
```diff
@@ -1,12 +1,36 @@
 ---
 layout: post
-title: Content with WinUI BusyIndicator control | Syncfusion
-description: Learn here all about Seven different animation types and features with Syncfusion WinUI BusyIndicator Control.
+title: AnimationType in WinUI BusyIndicator control | Syncfusion
+description: Learn here all about seven different animation types and features with Syncfusion WinUI BusyIndicator Control.
 platform: WinUI
```
Comment (Manivannan-E) [inline winui/BusyIndicator/AnimationType.md:5]: @Priyanka-Vijayasankar Learn here all about different animation types available in the Syncfusion WinUI BusyIndicator control.

Comment (Priyanka-Vijayasankar) [reply winui/BusyIndicator/AnimationType.md:5]: Changed the content as mentioned above.

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

File: winui/BusyIndicator/AnimationType.md [hunk 2]
```diff
@@ -1,12 +1,36 @@
 ---
 layout: post
-title: Content with WinUI BusyIndicator control | Syncfusion
-description: Learn here all about Seven different animation types and features with Syncfusion WinUI BusyIndicator Control.
+title: AnimationType in WinUI BusyIndicator control | Syncfusion
+description: Learn here all about different animation types available in the Syncfusion WinUI BusyIndicator control.
```
Comment (OnyimbiNelson) [inline winui/BusyIndicator/AnimationType.md:4]: description: Learn all about different animation types available in the Syncfusion WinUI BusyIndicator control here.

File: winui/BusyIndicator/AnimationType.md [hunk 3]
```diff
@@ -1,12 +1,36 @@
...
+
+SfBusyIndicator busyIndicator = new SfBusyIndicator();
+busyIndicator.IsActive = true;
+busyIndicator.AnimationType = BusyIndicatorAnimationType.DottedCircle;
+busyIndicator.Content = "Dotted circle";
+
+{% endhighlight %}
+{% endtabs %}
+
+The following gif image contains the types of animation in BusyIndicator.
```
Comment (OnyimbiNelson) [inline winui/BusyIndicator/AnimationType.md:33]: The following gif image contains the types of animation in BusyIndicator:

File: winui/Rating/Precision.md [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Precision with WinUI Rating control | Syncfusion
+title: Precision in WinUI Rating control | Syncfusion
 description: Learn here all about the precision feature in the Rating control to set the accuracy level such as full, half, and exact.
```
Comment (OnyimbiNelson) [inline winui/Rating/Precision.md:4]: description: Learn all about the precision feature in the Rating control to set the accuracy level such as full, half, and exact here.

File: winui/Rating/ToolTip.md [hunk 5]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: ToolTip with WinUI Rating control | Syncfusion
+title: ToolTip in WinUI Rating control | Syncfusion
 description: Learn here all about how to enable the tooltip and format the tooltip content in WinUI Syncfusion Rating (SfRating) control.
```
Comment (OnyimbiNelson) [inline winui/Rating/ToolTip.md:4]: description: Learn all about how to enable the tooltip and format the tooltip content in WinUI Syncfusion Rating (SfRating) control here.

File: winui/BusyIndicator/AnimationType.md (hunk 2 above)
Comment (Priyanka-Vijayasankar) [reply winui/BusyIndicator/AnimationType.md:4]: Changed the content as mentioned above.

File: winui/Rating/Precision.md (hunk 4 above)
Comment (Priyanka-Vijayasankar) [reply winui/Rating/Precision.md:4]: Changed the content as mentioned above.

**Pull request merged**